content/kubermatic/master/installation/add_seed_cluster/_index.en.md

```diff
@@ -0,0 +1,211 @@
++++
+title = "Add Seed Cluster"
+date = 2018-08-09T12:07:15+02:00
+weight = 30
+pre = "<b></b>"
++++
+
+This document describes how a new seed cluster can be added to an existing Kubermatic master cluster.
+
+### 1. Install Kubernetes cluster
+
+First, you need to install a Kubernetes cluster with some additional components. After the installation of
+Kubernetes you will need a copy of the `kubeconfig` to create a configuration for the new Kubermatic
+master/seed setup.
+
+To aid in setting up the seed and master clusters, we provide [KubeOne](https://github.com/kubermatic/kubeone/),
+which can be used to set up a highly-available Kubernetes cluster. Refer to the [KubeOne readme](https://github.com/kubermatic/kubeone/)
+and [docs](https://github.com/kubermatic/kubeone/tree/master/docs) for details on how to use it.
+
+Please take note of the [recommended hardware and networking requirements](../../requirements/cluster_requirements/)
+before provisioning a cluster.
+
+### 2. Install Kubermatic Dependencies
+
+When using Helm 2, install Tiller into the seed cluster first:
+
+```bash
+kubectl create namespace kubermatic
+kubectl create serviceaccount -n kubermatic tiller
+kubectl create clusterrolebinding tiller-cluster-role --clusterrole=cluster-admin --serviceaccount=kubermatic:tiller
+helm --service-account tiller --tiller-namespace kubermatic init
+```
+
+#### NodePort Proxy
+
+Kubermatic requires the NodePort Proxy to be installed in each seed cluster. The proxy is shipped as a
+[Helm](https://helm.sh) chart in the kubermatic-installer repository.
+
+As the NodePort Proxy Docker image is in a private registry, you need to configure the Docker Pull Secret for
+the Helm chart. You can re-use the `values.yaml` used during the installation or create new one and configure it like
+so:
+
+```yaml
+kubermaticOperator:
+  # insert the Docker authentication JSON provided by Loodse here
+  imagePullSecret: |
+    {
+      "auths": {
+        "quay.io": {....}
+      }
+    }
+```
+
+#### Cluster Backups
```

Comment:
A small illustration how the minio / user cluster backup cronjob works would help for an perspective of an end user to understand how this works